arenadata/Dockerfile

```diff
@@ -1,45 +1,53 @@
-FROM centos:centos7 as base
+FROM rockylinux:8.8 as base
 
 # Install some basic utilities and build tools
-RUN yum makecache && yum update -y ca-certificates && \
-    rpm --import https://mirror.yandex.ru/centos/RPM-GPG-KEY-CentOS-7 && \
-    yum -y install epel-release java-1.8.0-openjdk-devel && \
-    yum -y install git iproute net-tools openssh-server rsync sudo time vim wget unzip \
-                   ant-junit autoconf bison cmake3 flex gperf indent jq libtool gcc-c++ \
-                   krb5-server krb5-workstation xerces-c-devel file && \
-    yum clean all
+RUN dnf -y install 'dnf-command(config-manager)' && dnf config-manager --set-enabled devel && dnf makecache && dnf update -y ca-certificates && \
+    rpm --import https://mirror.yandex.ru/rockylinux/RPM-GPG-KEY-Rocky-8 && \
+    dnf -y install epel-release java-11-openjdk-devel && \
+    dnf -y install git iproute net-tools openssh-server rsync sudo time vim wget unzip \
+                   autoconf bison cmake3 flex gperftools indent jq libtool gcc-c++ \
+                   krb5-server krb5-workstation xerces-c-devel file netcat passwd lsof xz && \
+    dnf clean all
 
 # install all software we need
-RUN yum makecache && \
-    yum -y install python3-devel && \
-    yum -y install apr-devel bzip2-devel expat-devel libcurl-devel && \
-    yum -y install libevent-devel libuuid-devel libxml2-devel libyaml-devel libzstd-devel && \
-    yum -y install openssl-devel pam-devel readline-devel snappy-devel libuv-devel && \
-    yum -y install apache-ivy libicu perl-ExtUtils-Embed perl-Env perl-JSON && \
-    yum -y install perl-IPC-Run perl-Test-Base libxslt-devel openldap-devel && \
-    yum clean all
+RUN dnf makecache && \
+    dnf -y install python39-devel && \
```

Comment:
I tried to run tests in the container before this patch and got a error about python version. Why a different version was needed and how this issue was fixed?

Reply:
gpdb 7 migrated from python 2 to python 3

Reply:
I meant that the package `python3-devel` contained the wrong version of Python (`python3.6`) and now the package `python39-devel` has the correct version of Python.